docs/release_process.md

```diff
@@ -40,8 +40,18 @@ Let's make sure there are no migration conflicts with the previous release:
     $ git checkout master
     $ make migrate
```

Comment:
> This documentation version will be in the 0.22.0

And 0.22.0 won't be able to use `make test-data-migrations`, since the previous release didn't have the test data file.

That's why for 0.22.0 I preferred keeping both documentations. I already have a branch ready which removes it, and was thinking of doing it real soon after the 0.22.0 release, so that 0.23.0 only has the new documentation.